test/conformance/ingress/util.go

```diff
@@ -1123,6 +1132,13 @@ func RuntimeRequestWithExpectations(ctx context.Context, t *testing.T, client *h
 	opts ...RequestOption) *types.RuntimeInfo {
 	t.Helper()
 
+	t.Logf("delay of %d before doing the request \n", test.NetworkingFlags.RequestDelay)
```

Comment:
t.Logf already prints new lines I believe

Reply:
removed the new line